wp-content/themes/bpr2018/resources/templates/front-page.tpl.php

```diff
@@ -58,12 +58,14 @@ class="img-40 lazy-image"
                     </p>
       
                     <div class="post-author post-date font-size-20">
-                      <?php $author_id = $post->post_author; ?>
-                      <a href="<?php echo esc_url(get_author_posts_url($author_id)); ?>">
-                        <?php echo get_the_author_meta('display_name', $author_id); ?>
-                      </a>
+                      <span itemprop="author" hidden aria-hidden><?php the_author(); ?></span>
                       <?php
-                      if (get_the_date('', $id)) { echo ' | <time>' . get_the_date('', $id) . '</time>'; }
```

Comment:
I wonder if the `'', $id` matters

Reply:
hmmm, anyways, I just make a push again, this time, none of the get_the_date in that file has the weird parameters in the parentheses :)